mlflow/diviner.py

```diff
@@ -192,7 +222,60 @@ def save_model(
     _PythonEnv.current().to_yaml(str(path.joinpath(_PYTHON_ENV_FILE_NAME)))
 
 
-def load_model(model_uri, dst_path=None):
+def _save_model_fit_in_spark(diviner_model, path: str, **kwargs):
```

Comment:
Nit:

We can refactor code here to make it more clear:


```
def _save_diviner_model():
    if hasattr(diviner_model, "_fit_with_spark") and diviner_model._fit_with_spark:
        flavor_conf = {_SPARK_MODEL_INDICATOR: True}
        # _save_model_fit_in_spark code here...
        ...
    else:
        flavor_conf = {_SPARK_MODEL_INDICATOR: False}
        diviner_model.save(str(path.joinpath(_MODEL_BINARY_FILE_NAME)))



```

Reply:
Oh I see. When loading model we need to check `_SPARK_MODEL_INDICATOR` conf, current code is fine.

Reply:
+1 on the refactoring. It's much cleaner. TY!